src/static/js/modules/jquery/cf_inputSplit.js

```diff
@@ -190,3 +196,24 @@
     };
 
 })(jQuery);
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/bebb217d926085fc87d071fa3cb51ac548f9dc34